src/ctype/CType.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { SubmittableResult } from '@polkadot/api'
 import { TypeRegistry } from '@polkadot/types'
 import { Option } from '@polkadot/types/codec'
-
+import { submitSignedTx } from '../blockchain/Blockchain'
```

Comment:
You should only need the following.

`import { submitSignedTx } from '../blockchain'

Since we are accessing the index, if I am not mistaken.

Reply:
true, I recently added this to the index. Wasn't actually sure what to export from there, but this seems an essential function